hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCreateRequest.java

```diff
@@ -162,7 +163,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     OMClientResponse omClientResponse = null;
     try {
       // check Acl
-      checkBucketAcls(ozoneManager, volumeName, bucketName, keyName);
+      checkKeyAcls(ozoneManager, volumeName, bucketName, keyName,
```

Comment:
This requires a change in ozoneNativeAuthorizer, as this newly created Key does not have an entry in key table. (If this is created for the first time)

Reply:
And also our acceptance tests will fail.